scene/main/viewport.h

```diff
@@ -807,7 +809,7 @@ class SubViewport : public Viewport {
 
 	virtual Transform2D get_screen_transform_internal(bool p_absolute_position = false) const override;
 	virtual Transform2D get_popup_base_transform() const override;
-	virtual bool is_directly_attached_to_screen() const override;
+	virtual Viewport *get_section_rootviewport() const override;
```

Comment:
```suggestion
	virtual Viewport *get_section_root_viewport() const override;
```
`rootviewport()` is some odd spelling. Any specific reason why was it used?

Reply:
No specific reason. Will change it according to your suggestion.